src/SIL.XForge.Scripture/ClientApp/src/app/core/models/note-thread-doc.ts

```diff
@@ -101,7 +101,7 @@ export class NoteThreadDoc extends ProjectDataDoc<NoteThread> {
       return '';
     }
 
-    const iconDefinedNotes = this.notesInOrderClone(this.data.notes).filter(n => n.tagId != null);
+    const iconDefinedNotes = this.notesInOrderClone(this.data.notes).filter(n => n.tagId != null && !n.deleted);
```

Comment:
Rather than filtering on the front end, we should be filtering on the back end. We do this when fetching questions, where we query for only those that are not archived. I think we could do the same thing here.

Reply:
It is possible that this can be done, but it wouldn't be as simple. The questions collection can be filtered for not archived questions because that is a property on the Question model, but there isn't a defined way to query for note threads and then then filter for a specific subset of notes on a thread. So the simplest way to do it is just to filter it on the front end.